metals/src/main/scala/scala/meta/internal/decorations/SyntheticsDecorationProvider.scala

```diff
@@ -269,23 +265,70 @@ final class SyntheticsDecorationProvider(
     } yield docWithText
   }
 
-  private def toSymbolName(symbol: String, textDoc: TextDocument): String = {
-    if (symbol.isLocal)
-      textDoc.symbols
-        .find(_.symbol == symbol)
-        .map(_.displayName)
-        .getOrElse(symbol)
-    else symbol
+  private def localSymbolName(symbol: String, textDoc: TextDocument): String = {
+    textDoc.symbols
+      .find(_.symbol == symbol)
+      .map(_.displayName)
+      .getOrElse(symbol)
+  }
+
+  private def fullyQualifiedName(
+      symbol: String,
+      textDoc: TextDocument
+  ): String = {
+    if (symbol.isLocal) localSymbolName(symbol, textDoc)
+    else {
+      symbol
+        .replace("/", ".")
+        .stripSuffix(".")
+        .stripSuffix("#")
+        .stripPrefix("_empty_.")
+        .replace("#", ".")
+        .replace("()", "")
```

Comment:
it might be disambiguator with number like :`(+1)`

Reply:
Good catch, changed to .replaceAll(raw"\(\+?\d*\)", "")